verilog/tools/kythe/indexing_facts_tree_extractor_test.cc

```diff
@@ -122,16 +131,15 @@ TEST(FactsTreeExtractor, ParseErrorTest) {
       "final v;\n",         // syntax error
       "module unfinished",  // syntax error
   };
-  constexpr absl::string_view file_name = "verilog.v";
 
   for (const auto& code_text : code_texts) {
     int exit_status = 0;
-    bool parse_ok = true;
 
+    ScopedTestFile test_file(testing::TempDir(), code_text);
     const auto facts_tree =
-        ExtractOneFile(code_text, file_name, exit_status, parse_ok);
-    EXPECT_FALSE(parse_ok) << "code\n" << code_text;
-    EXPECT_EQ(exit_status, 1) << "code\n" << code_text;
+        ExtractFiles({std::string(test_file.filename())}, exit_status, "");
+    LOG(INFO) << facts_tree;
+    EXPECT_EQ(exit_status, 0) << "code\n" << code_text;
   }
```

Comment:
@fangism how should we test this now? 
we don't have parse_ok and exist_status now.
I think this isn't a suitable response to the user now we should return something more meaningful.